salt/client/ssh/shell.py

```diff
@@ -23,6 +23,10 @@
 SSH_PASSWORD_PROMPT_RE = re.compile(r'(?:.*)[Pp]assword(?: for .*)?:', re.M)
 KEY_VALID_RE = re.compile(r'.*\(yes\/no\).*')
 
+# Keep these in sync with ./__init__.py
+RSTR = '_edbc7885e4f9aac9b83b35999b68d015148caf467b78fa39c05f669c0ff89878'
```

Comment:
zomg, this is just a delimeter between STDOUT and STDERR... I recon, "RSTR" stands for "Random String, Truly Real".